src/lib.rs

```diff
@@ -28,7 +28,10 @@ use std::io::Write;
 
 pub use color::ColorType;
 
-pub use color::{Luma, LumaA, Rgb, Rgba, Bgr, Bgra};
+/// Structs that represent a single pixel of some color type.
+pub mod pixel {
+    pub use color::{Luma, LumaA, Rgb, Rgba, Bgr, Bgra};
+}
```

Comment:
These are not re-exports currently and also don't add clutter (structs are towards the bottom anyways). They are also likely among the most-used items. Moving them into a sepearte module only appears to complicate imports, I don't know what we gain from it.

Reply:
The fact that the listing of structs feels so far down in the docs is part of the clutter issue I'm trying to address. Either way, we don't need to decide on this now